aptos-move/e2e-move-tests/src/tests/aggregator_v2.data/pack/sources/aggregator_v2_test.move

```diff
@@ -0,0 +1,126 @@
+module 0x1::aggregator_v2_test {
+    use aptos_framework::aggregator_v2::{Self, Aggregator};
+    use aptos_std::table::{Self, Table};
+    use std::signer;
+
+    /// When checking the value of aggregator fails.
+    const ENOT_EQUAL: u64 = 17;
+
+    /// Resource to store aggregators. Each aggregator is associated with a
+    /// determinictic integer value, for testing purposes.
+    struct AggregatorStore has key, store {
+        aggregators: Table<u64, Aggregator>,
+    }
+    
+    /// Initializes a fake resource which holds aggregators.
+    public entry fun initialize(account: &signer) {
+        let aggregators = table::new();
+        let store = AggregatorStore { aggregators };
+        move_to(account, store);
+    }
+
+    public entry fun verify_copy_snapshot() {
```

Comment:
Seems like this and other `verify_` functions are never called?

Reply:
They are called in the `test_copy_snapshot` test.

Reply:
Cannot see the test in the diff?

Reply:
Found it :)